authorization/src/main/java/io/deephaven/auth/codegen/GenerateContextualAuthWiring.java

```diff
@@ -0,0 +1,111 @@
+package io.deephaven.auth.codegen;
```

Comment:
I personally would prefer if we kept it as a non-annotation processor but instead as a code generator that adheres to the protoc plugin API like this. It would technically work as an annotation processor, but we'd likely have to reflectively attempt to access the already-generated java types that came from protoc, so we'd have to do it later.

That said, I'd prefer if this lived in its own gradle module, as we'll eventually cook up the CI bits to make this happen in the same docker container that all the rest of the proto stuff goes on in.